chart/templates/deployment.yaml

```diff
@@ -151,6 +151,8 @@ spec:
 {{- end }}
         - name: IMPERATIVE_API_DIRECT
           value: "true"
+        - name: EXTENSION_API_APP_SELECTOR
```

Comment:
WDYT naming it to `IMPERATIVE_API_APP_SELECTOR` just to have some amount of consistency between that and `IMPERATIVE_API_DIRECT`?

Reply:
Oh yeah, thats definitely the right thing. I'll send up a fix for that in a few.